SQLRepository/CertificateRepositoryImpl.java

```diff
@@ -0,0 +1,79 @@
+package SQLRepository;
+
+import Entities.Certificate;
+import RepositoryInterfaces.CertificateRepository;
+import RepositorySpecification.CertificateSpecification;
+import RepositorySpecification.SelectAllCertificate;
+import RepositorySpecification.Specification;
+import RepositorySpecification.TagSpecification;
+import sqlconnection.SQLConnect;
+import CertificateRepository.CertificateMapper;
+
+import java.sql.Date;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class CertificateRepositoryImpl implements CertificateRepository {
+
+    static final String queryForAddRequest = "INSERT INTO certificate VALUES(?, ?, ?, ?, ?)";
```

Comment:
Where are Access Modifier? and if you declare a variable as static final, you should name it in upper case.

Reply:
like QUERY_FOR_ADD_REQUEST